app/src/main/res/values-zh-rHK/strings.xml

```diff
@@ -300,4 +304,7 @@
     <string name="simple_expert_settings">專家設定</string>
     <string name="copy_logs">將記錄複製到剪貼板</string>
     <string name="app_logo">應用程式標識</string>
-    </resources>
+    <string name="log_file">記錄檔案</string>
+    <string name="copying_logs_to_file">將記錄複製到檔案...</string>
```

Comment:
@rakekniven i am wondering whether you have a straight line for ellipsis characters (`…` instead of `...`)? I tried myself to use the ellipsis character at least in the english version. Is this something i should report to the translators if i notice it or do you think it's not worth the effort?

Reply:
@stefan-niedermann IMHO it is not worth the effort. As long as the source code contains the ellipsis it is ok for me. Haven't seen any UI broken by using threen single dots.